src/main/java/org/fundacion/jala/converter/controller/config/SwaggerConfig.java

```diff
@@ -26,24 +29,58 @@
 public class SwaggerConfig {
 
     /**
-     * Creates a Docket.
+     * Creates the API Docket.
      *
      * @return a Docket with Swagger configuration.
      */
     @Bean
     public Docket api() {
         return new Docket(DocumentationType.SWAGGER_2)
+                .groupName("API")
+                .securitySchemes(Arrays.asList(apiKey()))
                 .select()
                 .paths(PathSelectors.ant("/api/*"))
                 .apis(RequestHandlerSelectors.basePackage("org.fundacion.jala.converter"))
                 .build()
                 .apiInfo(apiDetails());
     }
 
+    /**
+     * Creates the authentication Docket.
+     *
+     * @return a Docket with Swagger configuration
+     */
+    @Bean
+    public Docket apiAuthentication() {
+        return new Docket(DocumentationType.SWAGGER_2)
+                .groupName("AUTHENTICATION")
+                .select()
+                .paths(PathSelectors.ant("/authenticate"))
+                .apis(RequestHandlerSelectors.basePackage("org.fundacion.jala.converter"))
+                .build()
+                .apiInfo(apiDetails());
+    }
```

Comment:
These lines could try to be up to 120 characters

Reply:
solved it.